bearer-token-example/get_session.py

```diff
@@ -0,0 +1,28 @@
+import contextlib
```

Comment:
I can move it somewhere else, but to be honest, I did not want it to be a test. I meant it to be an example of how you can write scripts in Python for Dataverse without the API key requirement. I guess someone with more knowledge of Python could work it out in a better way. I hoped that @JR-1991 would have a look at it, and maybe have some better ideas than selenium or cypress.

Reply:
I see. Yeah, examples would be good to put into the API guides.

WRT to good practices how to use this from within Python. For script usage, you should look into using the Device Auth Flow (preferred) - or - use a local webserver that can receive the auth codes (so enabling Auth Code Flow) - or - receive the tokens manually from the IdP and provide them at runtime.

Examples for this:
My co-worker @nheeb implemented the [Device Auth Flow with Github](https://github.com/softwarepub/hermes/blob/0651c7880f03dba0a3ff54282e0e1e38ffc99687/src/hermes/commands/init/oauth_github.py) for the HERMES init command.
He also [implemented the local server solution](https://github.com/softwarepub/hermes/blob/39eb860cf51dee3aa2507f256302d62e8fad0cb9/src/hermes/commands/init/oauth_process.py) before (but we removed it because we need public clients).